godotproject/RocketEnemy.gd

```diff
@@ -0,0 +1,89 @@
+extends KinematicBody2D
+
+const FLOOR = Vector2(0, -1)
+const SPEED = 15000
+const SPEED_INC = 125
+
+var velocity = Vector2(0, 0)
+
+var direction = Vector2(1, 0)
+var pastDirection = Vector2(1, 0)
+
+var attackDamage = 1
+var alpha = 0.1
+
+var timer
+
+enum STATE{
+	detected_player_e
+	attacking_player_e
+}
+
+var attackState = STATE.detected_player_e
+
+# Called when the node enters the scene tree for the first time.
+func _ready():
+	$Orientation/Area2D/CollisionShape2D.disabled = true
+	direction.x = 1
+	direction.y = 0
+	
+	timer = get_node("Timer")
+	timer.set_wait_time(0.4)
+	timer.connect("timeout", self, "_on_timer_timeout")
+	attackState = STATE.detected_player_e
+
+func _physics_process(delta):
+	#GetPlayerNode
+	if (attackState == STATE.attacking_player_e):
+		var level = get_parent().get_parent().name;
+		var playerPath = "/root/%s/Player" % level
+		var playerNode = get_node(playerPath)
```

Comment:
```suggestion
		var playerNode = $"../../Player"
```
Does this work? It should work the same but is a bit cleaner imo. Goes up two levels, then gets the node named "Player"